x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/PredictionFieldTypeTests.java

```diff
@@ -38,6 +38,8 @@ public void testTransformPredictedValueNumber() {
             is(nullValue()));
         assertThat(PredictionFieldType.NUMBER.transformPredictedValue(1.0, "foo"), equalTo(1.0));
         assertThat(PredictionFieldType.NUMBER.transformPredictedValue(1.0, null), equalTo(1.0));
+        long expected = randomLong();
```

Comment:
Could you add tests for:
1. an empty string
2. a string that is not a number?

Reply:
There is already a test for 2. See line 39.

I will add a test for 1